### PR TITLE
Check for org_id

### PR DESCRIPTION
**Pull request by JJedrasik**

### CODE REVIEW

**Review by inaimathi — CHANGES_REQUESTED** [review]

File: src/pycroservice/core.py [hunk 1]
```diff
@@ -98,6 +98,8 @@ def _scope_check(token, scopes, params):
     if intersect and {s for s in intersect if s.startswith("xorg_")}:
         return True, None
 
+    if "org_id" not in params: 
+        return True, "None"
```
Comment (inaimathi) [inline src/pycroservice/core.py:102]: Not the string none. Just `None`.

File: src/pycroservice/core.py [hunk 2]
```diff
@@ -121,10 +123,10 @@ def decorator(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
             token = _reqTok(request)
-
+
```
Comment (inaimathi) [inline src/pycroservice/core.py:126]: Remove weird whitespace

File: src/pycroservice/core.py [hunk 3]
```diff
@@ -121,10 +123,10 @@ def decorator(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
             token = _reqTok(request)
-
+            
             if token is None:
                 return jsonError("Token is missing", 401)
-
+
```
Comment (inaimathi) [inline src/pycroservice/core.py:129]: Ditto

**Review by inaimathi — CHANGES_REQUESTED** [review]

File: src/pycroservice/core.py (hunk 3 above)
Comment (JJedrasik) [reply src/pycroservice/core.py]: I'll make `black` part of the future pipeline as well

**Review by inaimathi — APPROVED** [review]

**Pull request merged**